Modify documentation for stable storage (#29012)
kubernetes/website: content/en/docs/concepts/workloads/controllers/statefulset.md

### Stable Storage

> Per each StatefulSet triggered pod Kubernetes creates a PersistentVolumeClaim object for each VolumeClaimTemplates entry defined in the StatefulSet object.In the nginx example above, each Pod will receive a single PersistentVolume

Reviewer: - `pod` -> `Pod`
- Added line breaks
  - (As-is: There is no space between `object.` and `In`)

```suggestion
Per each StatefulSet triggered Pod Kubernetes creates a PersistentVolumeClaim object 
for each VolumeClaimTemplates entry defined in the StatefulSet object.
In the nginx example above, each Pod will receive a single PersistentVolume
```

Author: @jihoon-seo Thanks for the review. I will be updating this

Author: @jihoon-seo Addressed the Changes. PTAL!

Author: Hi @jihoon-seo all the objectives for the required changes are addressed, Please let me know for any further changes if required.
Thanks

> Per each StatefulSet triggered Pod Kubernetes creates a PersistentVolumeClaim object for each VolumeClaimTemplates entry defined in the StatefulSet object. In the nginx example above, each Pod will receive a single PersistentVolume

Reviewer 2: The grammar in the original issue wasn't fantastic. Try...

```suggestion
For each VolumeClaimTemplate entry defined in a StatefulSet, each Pod receives one PersistentVolumeClaim. In the nginx example above, each Pod receives a single PersistentVolume
```

Author: Hi i have addressed the changes PTAL!

Author: @celestehorgan Please let me know if any other changes are required in this PR.

Thanks!

Author: Hi @jihoon-seo Could you Please take a look here. I have addressed the changes as suggested
